app/assets/javascripts/components/schools/schools_page.es6.jsx

```diff
@@ -35,6 +26,13 @@ class SchoolsPage extends Component {
     };
   }
 
+  componentDidMount() {
+    resolve = (response) => {
+      this.setState({ schools: JSON.parse(response) })
+    };
+    Requester.get(ApiConstants.schools.index, resolve);
+  }
```

Comment:
'componentDidMount' is a lifecycle method invoked by React when this element is attached to the browser page.
Once this is invoked, we request schools from our newly-created api - then we update our state with the JSON passed down. Essentially, when the page first pops open we won't have any data, but once the request comes back we will (you might not even notice since it's so fast).